tests/robot/apps/automation/ansible/tower/hosts.robot

```diff
@@ -0,0 +1,40 @@
+*** Settings ***
+Documentation       Check the hosts mode with api custom mode
+
+Resource            ${CURDIR}${/}..${/}..${/}..${/}..${/}..${/}resources/import.resource
+
+Suite Setup         Start Mockoon    ${MOCKOON_JSON}
+Suite Teardown      Stop Mockoon
+Test Timeout        120s
+
+
+*** Variables ***
+${MOCKOON_JSON}     ${CURDIR}${/}ansible_tower.json
+
+${CMD}              ${CENTREON_PLUGINS}
+...                 --plugin=apps::automation::ansible::tower::plugin
+...                 --custommode=api
+...                 --hostname=${HOSTNAME}
+...                 --username=username
+...                 --password=password
+...                 --port=${APIPORT}
+
+
+*** Test Cases ***
+Hosts ${tc}
+    [Documentation]    Check the number of returned hosts
+    [Tags]    apps    automation    ansible    service-disco
+    ${command}    Catenate
+    ...    ${CMD}
+    ...    --mode=hosts
+    ${output}    Run    ${command}
+    ${output}    Strip String    ${output}
+    Should Be Equal As Strings
```

Comment:
The ideal refacto:
Create a new keyword: `Ctn Run Command And Check Result As Strings`
It will Run the command, Strip String the result, compare the output and return the error message